src/test/java/io/jenkins/plugins/opentelemetry/JenkinsOtelPluginIntegrationTest.java

```diff
@@ -325,6 +332,8 @@ private void assertGitCredentials(String jobName, String globalCredentialId, Str
 
         Attributes attributes = gitNode.get().getData().spanData.getAttributes();
         MatcherAssert.assertThat(attributes.get(JenkinsOtelSemanticAttributes.GIT_USERNAME), CoreMatchers.is(gitUserName));
+        MatcherAssert.assertThat(attributes.get(JenkinsOtelSemanticAttributes.JENKINS_STEP_PLUGIN_NAME), CoreMatchers.is(CoreMatchers.notNullValue()));
+        MatcherAssert.assertThat(attributes.get(JenkinsOtelSemanticAttributes.JENKINS_STEP_PLUGIN_VERSION), CoreMatchers.is(CoreMatchers.notNullValue()));
```

Comment:
Unfortunately the classloader in the JenkinsRule does not work as expected, so Ideally the assertion should be `is("git")` but in the UTs the Jenkins context cannot retrieve the Git plugin wrapper.

Reply:
I've tried with the `withPlugin` annotation but same issue :/